apps/ui-layout/registry/components/linear-modal/center-content.tsx

```
@@ -51,14 +51,14 @@ export default function App() {
                   borderRadius: '12px',
                 }}
-                className='flex w-full h-72 border dark:bg-black bg-white flex-col overflow-hidden'
+                className='flex w-full h-72 border bg-background flex-col overflow-hidden'
               >
                 <DialogImage src={item.url} alt='' className='h-full w-full object-cover' />
                 <div className='flex grow flex-row items-end justify-between p-3 pt-4 text-primary'>
                   <div>
                     <DialogTitle className='text-xl '>{item.title}</DialogTitle>
                   </div>
-                  <button className='absolute bottom-2 right-2 p-2 bg-neutral-200 dark:bg-neutral-900 rounded-lg'>
+                  <button className='absolute bottom-2 right-2 p-2 bg-muted rounded-lg'>
```

Biome (2.4.16)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)